Sources/SwiftProtobuf/StringUtils.swift

```diff
@@ -22,14 +22,19 @@ import Foundation
 // This is painfully slow but seems to work correctly on every platform.
 // We currently only use it on Linux.  See below.
 fileprivate func slowUtf8ToString(bytes: UnsafePointer<UInt8>, count: Int) -> String? {
-    var s = ""
-    let buffer = UnsafeBufferPointer<UInt8>(start: bytes, count: count)
-    var bytes = buffer.makeIterator()
-    var utf8Decoder = UTF8()
+    // Prepare to iterate through the input
+    let buffer = UnsafeBufferPointer(start: bytes, count: count)
+    var it = buffer.makeIterator()
+    // Using this decoder
```

Comment:
I suggest renaming `scalars` to `output` and dropping your one-line comments here.